Sources/SwiftSyntax/generated/syntaxNodes/SyntaxDeclNodes.swift

```diff
@@ -3778,7 +3778,7 @@ public struct InitializerDeclSyntax: DeclSyntaxProtocol, SyntaxHashable {
     }
   }
   
-  /// The arguments of the initializer. While the function signature allows specifying an return clause, doing so is not semantically valid.
+  /// The arguments of the initializer. While the function signature allows specifying a return clause, doing so is not semantically valid.
```

Comment:
This file is being generated. You need to also fix the typo here:

https://github.com/apple/swift-syntax/blob/1d36fc2b6a923a1ae79eede12bdd977632e33815/CodeGeneration/Sources/SyntaxSupport/DeclNodes.swift#L1344